shared/clinical-code-sets/conditions/autism-spectrum-disorder/1/autism-spectrum-disorder.emis.txt

```diff
@@ -18,4 +18,87 @@
 ^ESCTPE307849	Pervasive developmental disorder (PDD)
 ^ESCTPR843868	Pre-school Autism Communication Therapy
 ^ESCTRE846354	Referral to autism assessment service
-EGTON219	Suspected autism
\ No newline at end of file
+EGTON219	Suspected autism
+
```

Comment:
not sure if this gap of three empty lines will affect the scripts running?

Reply:
I don't think it will, but probably good to fix anyway. Thanks.